src/genie/libs/parser/ios/show_vlan_internal_usage.py

```diff
@@ -0,0 +1,65 @@
+'''
+Author: Mehdi Cherifi
+Twitter: https://twitter.com/LocketKeepsake
+Github: https://github.com/cherifimehdi
+
+'''
+import re
+
+# Metaparser
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Any
+
+
+# ====================================================
+#  Schema for 'show vlan internal usage' command
+# ====================================================
+class ShowVlanInternalUsageSchema(MetaParser):
+    """Schema for:
+        show vlan internal usage"""
+
+    schema = {
+        'Internal Vlan': {
```

Comment:
Key names should all be lowercase with spaces replaced by underscores
```suggestion
        'internal_vlan': {
```